lib/imageDisplay.ml

```diff
@@ -122,12 +122,10 @@ let print_portrait conf base p =
     [Rem] : Ne pas utiliser en dehors de ce module.                           *)
 let print_source conf f =
   let fname = if f.[0] = '/' then String.sub f 1 (String.length f - 1) else f in
-  if fname = Filename.basename fname then
```

Comment:
why was this check unnecessary?

Reply:
This check prevent the use of sub-folders for images and source files.
Secure.open ensures that the opened file is within the -bd sub-space.
Adding one or several ../ in the filename to access files outside images or src triggers an error